### ISSUE TITLE
Addition to list of tools?

### ISSUE THREAD

**Issue by jamesrco** [RASpicer/MetabolomicsTools#2]
Hi there --
I am wondering how one might add another tool to your (awesome!) list?
Thanks,
Jamie Collins

**Comment by RASpicer** [comment]
Hi Jamie,
If you'd like to add a tool I can add you as a collaborator? Or you can send me the info and I'll add it, whatever you prefer.
Best,
Rachel

**Comment by rsalek** [comment]
+1

**Comment by jamesrco** [comment]
Hi Rachel!
I don't mind adding it... pull request easiest way you think?
Jamie

**Comment by RASpicer** [comment]
Yes I think a pull request would be the easiest way to do it. Send it whenever you're ready.
Best,
Rachel

**Comment by jamesrco** [comment]
@RASpicer: Great, thanks. Should I just create a new .md file for the tool I want to add and then do a PR on that -- or should I try and update the main table and necessary Wiki entires also? Not sure what your workflow looks like and don't want to interfere with it. Also: What is the reference for the level of annotation? Is it the Sumner et al. 2007 paper, or some other rubric?

**Comment by RASpicer** [comment]
@jamesrco sorry for the slow reply.
Yes please just create a new .md file and I'll update the rest. 
The annotation levels are based off Schymanski et al. 2014 criteria - available here (http://pubs.acs.org/doi/abs/10.1021/es5002105). Please let me know if you have any more questions!
